### PR TITLE
Improve logging infrastructure

### PR DESCRIPTION
**Pull request by kumaya**

### CODE REVIEW

File: atlas-db-controller/databaseSchemaResource.go [hunk 1]
```diff
@@ -231,7 +212,7 @@ func (c *Controller) updateDatabaseSchemaStatus(key string, schema *atlas.Databa
 
 	_, err := c.atlasclientset.AtlasdbV1alpha1().DatabaseSchemas(schema.Namespace).Update(schemaCopy)
 	if err != nil {
-		runtime.HandleError(fmt.Errorf("error updating status to '%s' for database schema '%s': %s", state, key, err))
+		logrus.Warningf("error updating status to '%s' for database schema '%s': %s", state, key, err)
```
Comment (jkraj) [inline atlas-db-controller/databaseSchemaResource.go]: why runtime.HandleError is not used here?

File: atlas-db-controller/databaseServerResource.go [hunk 2]
```diff
@@ -42,24 +41,23 @@ func (c *Controller) syncServer(key string) error {
 	if err != nil {
 		// The DatabaseServer resource may no longer exist, in which case we stop processing.
 		if errors.IsNotFound(err) {
-			runtime.HandleError(fmt.Errorf("databaseserver '%s' in work queue no longer exists", key))
+			logrus.Warningf("databaseserver '%s' in work queue no longer exists", key)
```
Comment (jkraj) [inline atlas-db-controller/databaseServerResource.go]: same here?

File: atlas-db-controller/databaseSchemaResource.go (hunk 1 above)
Comment (kumaya) [reply atlas-db-controller/databaseSchemaResource.go]: `runtime.HandleError()` is simply printing the error messages which we are doing anyway with logging statements. So removing them

File: atlas-db-controller/databaseServerResource.go (hunk 2 above)
Comment (kumaya) [reply atlas-db-controller/databaseServerResource.go]: same as above

File: atlas-db-controller/databaseSchemaResource.go [hunk 3]
```diff
@@ -27,33 +26,33 @@ func (c *Controller) enqueueDatabaseSchema(obj interface{}) {
...
 	if err != nil {
 		if errors.IsNotFound(err) {
-			runtime.HandleError(fmt.Errorf("schema '%s' in work queue no longer exists", key))
+			logrus.Warningf("schema '%s' in work queue no longer exists", key)
 			return nil
 		}
 		return err
 	}
-	glog.V(4).Infof("Schema Spec: %v", schema.Spec)
+	logrus.Infof("Schema Spec: %v", schema.Spec)
```
Comment (jkraj) [inline atlas-db-controller/databaseSchemaResource.go]: should be debug.

File: atlas-db-controller/databaseServerResource.go [hunk 4]
```diff
@@ -3,37 +3,36 @@ package main
...
-	"k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/tools/cache"
 )
 
 func (c *Controller) enqueueDatabaseServer(obj interface{}) {
 	var object metav1.Object
 	var ok bool
 	if object, ok = obj.(metav1.Object); !ok {
-		glog.Info("not enqueue server object")
+		logrus.Info("not enqueue server object")
```
Comment (jkraj) [inline atlas-db-controller/databaseServerResource.go]: debug

File: atlas-db-controller/databaseServerResource.go [hunk 5]
```diff
@@ -42,24 +41,23 @@ func (c *Controller) syncServer(key string) error {
...
+			logrus.Warningf("databaseserver '%s' in work queue no longer exists", key)
 			return nil
 		}
 		return err
 	}
 
 	p := server.ActivePlugin(s)
 
-	glog.V(4).Infof("DatabaseServer %s has plugin type: %T", key, p)
+	logrus.Infof("DatabaseServer %s has plugin type: %T", key, p)
```
Comment (jkraj) [inline atlas-db-controller/databaseServerResource.go]: Debug

File: atlas-db-controller/databaseServerResource.go [hunk 6]
```diff
@@ -214,7 +205,7 @@ func (c *Controller) syncService(key string, s *atlas.DatabaseServer) error {
 					},
 				},
 			}
-			glog.V(4).Info("Creating service with type ClusterIP")
+			logrus.Info("Creating service with type ClusterIP")
```
Comment (jkraj) [inline atlas-db-controller/databaseServerResource.go]: Debug

File: atlas-db-controller/databaseServerResource.go [hunk 7]
```diff
@@ -223,14 +214,13 @@ func (c *Controller) syncService(key string, s *atlas.DatabaseServer) error {
 					ExternalName: s.Spec.DBHost,
 				},
 			}
-			glog.V(4).Info("Creating service with type ExternalName")
+			logrus.Info("Creating service with type ExternalName")
```
Comment (jkraj) [inline atlas-db-controller/databaseServerResource.go]: Debug

File: atlas-db-controller/databaseServerResource.go [hunk 8]
```diff
@@ -288,7 +276,7 @@ func (c *Controller) syncPodServer(p plugin.PodPlugin, key string, s *atlas.Data
 		   // Update the pod resource to match the spec
 		   diffs := p.DiffPod(s)
 		   if diffs != "" {
-		       glog.V(4).Infof("DatabaseServer %s needs update: %s", diffs)
+		       logrus.Infof("DatabaseServer %s needs update: %s", diffs)
```
Comment (jkraj) [inline atlas-db-controller/databaseServerResource.go]: Debug

File: atlas-db-controller/databaseSchemaResource.go [hunk 9]
```diff
@@ -111,23 +107,20 @@ func (c *Controller) syncSchema(key string) error {
 			if err != nil {
 				if errors.IsNotFound(err) {
 					msg := fmt.Sprintf("waiting to get sourceURL for schema `%s` from secret `%s`", key, secretName)
-					glog.Info(schemaStatusMsg)
+					logrus.Info(schemaStatusMsg)
```
Comment (jkraj) [inline atlas-db-controller/databaseSchemaResource.go]: Debug

**Pull request merged**